chat-demo-android/src/main/kotlin/com/twilio/chat/demo/views/ChannelViewHolder.kt

```diff
@@ -54,21 +54,21 @@ class ChannelViewHolder : SettableViewHolder<ChannelModel>, AnkoLogger {
         channel.getUnconsumedMessagesCount(object : CallbackListener<Long>() {
             override fun onSuccess(value: Long?) {
                 Log.d("ChannelViewHolder", "getUnconsumedMessagesCount callback")
-                unconsumedMessagesCount.text = "Unread " + value!!.toString()
+                unconsumedMessagesCount.text = "Unread " + (value?.toString() ?: 0)
```

Comment:
`null` doesn't mean 0 unread messages here and in other places. It means "unknown", or "Consumption horizon hasn't been set yet".  In `6.0.0` me made it nullable especially to split these two to cases. So I'd suggest smth like:
```
"Unread: ${value ?: "unknown"}"
```

Reply:
`null` means either **ALL messages are unread** _or_ **we do not care**.